frameworks/autosklearn/exec.py

```diff
@@ -50,45 +51,64 @@ def run(dataset, config):
     y_train = dataset.train.y_enc
     predictors_type = dataset.predictors_type
     log.debug("predictors_type=%s", predictors_type)
-    # log.info("finite=%s", np.isfinite(X_train))
 
     training_params = {k: v for k, v in config.framework_params.items() if not k.startswith('_')}
 
     n_jobs = config.framework_params.get('_n_jobs', config.cores)
     ml_memory_limit = config.framework_params.get('_ml_memory_limit', 'auto')
-    ensemble_memory_limit = config.framework_params.get('_ensemble_memory_limit', 'auto')
 
-    # when memory is large enough, we should have:
-    # (cores - 1) * ml_memory_limit_mb + ensemble_memory_limit_mb = config.max_mem_size_mb
+    constr_params = {}
+    fit_extra_params = {}
+
     total_memory_mb = utils.system_memory_mb().total
     if ml_memory_limit == 'auto':
-        ml_memory_limit = max(min(config.max_mem_size_mb,
-                                  math.ceil(total_memory_mb / n_jobs)),
-                              3072)  # 3072 is autosklearn defaults
-    if ensemble_memory_limit == 'auto':
-        ensemble_memory_limit = max(math.ceil(ml_memory_limit - (total_memory_mb - config.max_mem_size_mb)),
-                                    math.ceil(ml_memory_limit / 3),  # default proportions
-                                    1024)  # 1024 is autosklearn defaults
-    log.info("Using %sMB memory per ML job and %sMB for ensemble job on a total of %s jobs.", ml_memory_limit, ensemble_memory_limit, n_jobs)
+        ml_memory_limit = max(
+            min(
+                config.max_mem_size_mb / n_jobs,
+                math.ceil(total_memory_mb / n_jobs)
+            ),
+            3072  # 3072 is autosklearn default and we use it as a lower bound
+        )
```

Comment:
this block seems to be used only for version > 0.11, maybe it should be included in the if clause below.
Also, it looks very similar to lines 82-86 below with one difference:
```
L67: config.max_mem_size_mb / n_jobs,
L84: config.max_mem_size_mb
```
L84 looks like a mistake (it was probably a mistake before your changes) if we want to fulfill the equation L81.

Reply:
Thanks for pointing this out. Yes, line 84 was a mistake that was there before which I forgot to update. But I think the lines in 82-86 can safely be dropped as they are the same as the new lines 65-72.